tools/azure-sdk-tools/devtools_testutils/azure_testcase.py

```diff
@@ -268,7 +268,8 @@ def get_preparer_resource_name(self, prefix):
     @staticmethod
     def await_prepared_test(test_fn):
         """Synchronous wrapper for async test methods. Used to avoid making changes
-        upstream to AbstractPreparer (which doesn't await the functions it wraps)
+        upstream to AbstractPreparer, which only awaits async tests that use preparers.
+        (Add @AzureTestCase.await_prepared_test decorator to async tests without preparers)
         """
 
```

Comment:
```suggestion
# Note: This will only be needed so long as we maintain unittest.TestCase in our test-class inheritance chain.
```
Since we both had the same question when we saw the bigger picture, maybe worth leaving a note here for our future selves/others?